man/build_ord_obs.Rd

```diff
@@ -0,0 +1,42 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/postprocess.R
+\name{build_ord_obs}
+\alias{build_ord_obs}
+\title{Build the ord_obs \code{data.table}.}
+\usage{
+build_ord_obs(obs, max_delay, internal_timestep, timestep, nowcast = NULL)
+}
+\arguments{
+\item{obs}{Observations as pulled from nowcast$latest[\link{1}].}
```

Comment:
I think this is the link problem (will go and read the non md but my expectation is that this code is missing back ticks. I would be interested to know if your LLM of choice could have caught this (because language style is something they are good at and the human eye is not)

Reply:
update after reading the raw roxygen and yes I think this is the problem